frontend/src/views/teams.js

```diff
@@ -395,8 +431,34 @@ export function EditTeam(props) {
           type="members"
           memberJoinTeamError={memberJoinTeamError}
           setMemberJoinTeamError={setMemberJoinTeamError}
+          enableTeamsIntegration={ENABLE_OSM_TEAMS_INTEGRATION}
+          disableEdit={osmTeamsId}
         />
         <div className="h1"></div>
+        {ENABLE_OSM_TEAMS_INTEGRATION && (osmTeamsId || (members.length < 1 && managers.length < 2)) && (
+          <div className="mb3">
+            <TeamSync
+              osmTeamsId={osmTeamsId}
+              setOsmTeamsId={setOsmTeamsId}
+              setManagers={setManagers}
+              setMembers={setMembers}
+              managers={managers}
+              members={members}
+              tmTeamId={props.id}
```

Comment:
I wanted to clarify that you must have meant just `id` instead of team ID since the team ID would be available from URL parameters and not from props.

Reply:
thanks for spotting it